src/MediaType.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Swagger package.
+ *
+ * (c) EXSyst
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace EXSyst\OAS;
+
+use EXSyst\OAS\Collections\Examples;
+
+final class MediaType extends AbstractObject
+{
+    use ExtensionPart;
+
+    /** @var Schema */
+    private $schema;
+
+    /** @var Examples|Example[]|Reference[] */
+    private $examples;
+
+    /** @var mixed */
+    private $example;
+
+    public function __construct(array $data = [])
+    {
+        $this->schema = isset($data['schema']) ? referenceOr(Schema::class, $data['schema']) : null;
+        $this->examples = new Examples($data['examples'] ?? []);
+        $this->example = $data['example'] ?? null;
```

Comment:
What about only supporting `examples` and use `example` in `export` if there's only one example provided?

Reply:
ok